magento/magento2
# 10128: New Orders not being saved to order grid

New Orders not being saved to order grid

### Fixed Issues (if relevant)

1. magento/magento2#10128: New Orders not being saved to order grid

### Manual testing scenarios
1. Go to Storefront
2. Add Product to the Card as Guest
3. Go to Checkout
4. Fill all required fields
5. Choose internal payment method "Check/Money order"
6. Click Place Order button
7. Log in to Admin
8. Go to Dashboard

Actual result:
-Dashboard shows orders as "Guest"
Expected result:
-Dashboard should show order with customer name

### Contribution checklist
